services/release-creator.js

```diff
@@ -101,7 +101,7 @@ function ReleaseCreator(argv, options = {}) {
     changes.splice(index, 0, `\n## RELEASE ${withVersion ? `${version} ` : ''}- ${today}`);
     const newChanges = changes.join('\n');
 
-    const commitMessage = withVersion ? `Release ${version}` : `Release - ${today}`;
+    const commitMessage = `chore(release): ${withVersion ? version : today}`;
```

Comment:
[MISC] This is intended to reproduce the behavior of the `forest-express` repository. I just got rid of the `[skip ci]` as the conventional commits are not fully operational (.ie auto-release etc.)